frontend/src/components/RatingTableValue.tsx

```diff
@@ -48,6 +63,9 @@ const TableValueRatingRow: RatingRow = ({ rating, style, user, onEdit }) => {
             user.representativeFor?.some(
               ({ id }) => id === rating.forCustomer,
             ) && <EditButton fontSize="medium" onClick={onEdit} />}
+          {hasEditOthersPermission && (
+            <DeleteButton onClick={(e) => onDelete(e, rating)} />
```

Comment:
This does not necessarily need to be changed, but consider changing the onDelete to `onDelete: () => void;`
```suggestion
            <DeleteButton onClick={onDelete} />
```

The Row's onDelete would turn to `onDelete={openDeleteModal(ratings[index])}`

I suggest this, since I don't see why the event default functionality or propagation would need to be cancelled in this case and the rating can be accessed in the RatingTable.

Reply:
> The Row's onDelete would turn to onDelete={openDeleteModal(ratings[index])}

Everything else worked fine, but with this line it seemed that I needed to wrap that function.

`onDelete={() => openDeleteModal(ratings[index])}`

Otherwise, it will complain about "Type 'void' is not assignable to type '() => void'.ts(2322)"

Not sure whats that about.

Reply:
Right, cause without the wrapping, its trying to pass the returned value of `openDeleteModal(ratings[index])` instead of the function itself.

Reply:
You are correct! I'm sorry for any extra work caused by the bad example

Reply:
All good 👍

Made me read couple of articles about function passing in React and now I got better understanding of whats going on.